Source/relay/TourGuide/Items of the Month/2014/Speakeasy.ash

```diff
@@ -26,8 +26,7 @@ void IOTMSpeakeasyGenerateResource(ChecklistEntry [int] resource_entries)
     string [int][int] options;
     
     options.listAppend(listMake("<strong>Drink</strong>", "<strong>Size</strong>", "<strong>Description</strong>"));
-    if (CounterLookup("Semi-rare").CounterIsRange())
-        options.listAppend(listMake("Lucky Lindy", "1", "Semi-rare number"));
+    options.listAppend(listMake("Lucky Lindy", "1", "Get Lucky!"));
```

Comment:
please modify size while we're updating this; lucky lindy was upped to a potency of 6 in the lucky changes in 2022 new years. it's kind of funny that in all the unrestricted runs i've done since 2022 new years i never noticed this bug because this vanished due to the counter lookup being bullshit now lol

Reply:
Fixed.